Remove ambiguity in params allignment

srt32 opened the pull request:
Why:

* both the below options were previouls good but we've had a number of
  discussions around what we want. This change lands on a single one
  (pushing code as far left as possible).

This change addresses the need by:

* removing the ambiguity and stating that when method params are broken
  onto multiple lines put one on each line and indent once from the
  method declaration. This change makes it easier to scan the list.

ryanclark2 commented:
👍 An alternative edit would be to leave it as bad so there is even less ambiguity.

Commit:
Remove ambiguity in params allignment

Why:

* both the below options were previouls good but we've had a number of
  discussions around what we want. This change lands on a single one
  (pushing code as far left as possible).

This change addresses the need by:

* removing the ambiguity and stating that when method params are broken
  onto multiple lines put one on each line and indent once from the
  method declaration. This change makes it easier to scan the list.

srt32 commented:
Fantastic idea. Updated.
